client/ayon_equalizer/api/host.py

```diff
@@ -323,9 +323,16 @@ def install(self) -> None:
         register_loader_plugin_path(LOAD_PATH)
         register_creator_plugin_path(CREATE_PATH)
 
-        heartbeat_interval = os.getenv("AYON_TDE4_HEARTBEAT_INTERVAL") or 10
+        try:
+            heartbeat_interval = int(
+                os.getenv("AYON_TDE4_HEARTBEAT_INTERVAL")) or 100
+        except ValueError:
```

Comment:
`os.getenv("AYON_TDE4_HEARTBEAT_INTERVAL")` can return `None` in that case `int` conversion raises `TypeError`.
```suggestion
        except (ValueError, TypeError):
```